agent/README.md

```diff
@@ -455,147 +455,7 @@ information.
-as well as [this StackOverflow answer][so-java-exec-answer].  
-**NOTE**: If this is not working with `CMD` or if you are generally interested in the difference between `CMD` and `ENTRYPOINT` see [this StackOverflow answer](https://stackoverflow.com/a/39408777).
-
-
-Otherwise you can simply configure the Dockerfile to start the java process with either CMD or ENTRYPOINT as follows:
-- `CMD ["java", ...]`
-- `ENTRYPOINT ["java", ...]`
-
-
-#### Start the java process inside of a ENTRYPOINT script 
-If the java process is the last call in your script, then you can simply use `exec`.
-This will give the java process the PID 1.
-```bash
-# !/bin/sh
-# Preparations ...
-exec java -javaagent:/agent/teamscale-jacoco-agent.jar=config-file=/agent/jacoco.conf -jar /app/app.jar
-```
-
-##### More complex
-If you have an entrypoint script for your Docker container which does more than a few preparations and then starting the java application,
-you can push the java application into the background and wait for it to finish, trap any shut-down signal and
-pass them the java process.
-
-Here is an example script how such an entrypoint.sh might look like. For more in-depth information take a look at [this article][signal-trapping]
-
-```bash
-#!/bin/sh                                                                                           
-
-# Forward SIGTERM                                                                                   
-_term() {
-  echo "Forward SIGTERM to $pid"
-  kill -TERM "$pid"
-  wait $pid
-}
-
-# Forward SIGINT
-_int() {
-  echo "Forward SIGINT to $pid"
-  kill -INT "$pid"
-  wait $pid
-}
-
-# Capture SIGTERM and SIGINT so we can forward them to the java application                                 
-trap _term TERM                                                                                     
-trap _int INT                                                                                       
-
-# Push java in the background with '&' at the end 
-java -javaagent:/agent/teamscale-jacoco-agent.jar=config-file=/agent/jacoco.conf -jar /app/app.jar & 
-pid=$! # Get the PID of the most recent background command
-
-# Here: Code which should be executed after the java-app has started
-# Be careful with long running commands here because if this script receives a signal while other 
-# non-background commands are running it will not trap any signals until those commands are done. 
-# It will simply wait for them to finish and in case of docker it will send a SIGKILL after 
-# a 10 second time-out which will end everything directly and any coverage will be lost.
-
-wait $pid # At some point we need to wait for the java process before the script ends
-wait $pid # Second wait necessary because the first wait will exit before the trapping of the signals
-
-# Here: Code which should be executed after the java-app is finished or has been terminated
-```
```

Comment:
This is missing from the documentation on [docs.teamscale.com](https://docs.teamscale.com/howto/recording-test-coverage-for-java-with-docker). Since this information might be difficult to to find again, I'd like to preserve it somewhere. Should we add it to the new documentation?

Reply:
Nvm, saw your comment too late on Jira.